src/qibosoq/components.py

```diff
@@ -1,9 +1,8 @@
 """Various helper objects."""
 
-from collections.abc import Iterable
 from dataclasses import dataclass, field
 from enum import IntEnum, auto
-from typing import List, Union, overload
+from typing import Iterable, List, Union, overload
```

Comment:
It was fine to import `Iterable` from `collections.abc`, since py3.9 it is the recommended way.
(of course, fine unless we have to support py3.8)

Reply:
In the same spirit you can use `list` instead of `List`

Reply:
On the board there is python 3.8.2

Reply:
Ah, ok